### PR TITLE
Documentation ag

### PR DESCRIPTION
**Pull request by grandrea**
- fixed small typos
- moved section on editing gui
- added running from command line
- added image

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,14 @@ xiSEARCH is a search engine for crosslinking mass spectrometry (crosslinking MS)
...
 
-The xiSEARCH algorithm is described in detail in [Mendez, Fischer *et al.* Mol. Sys. Bio. 2019](https://www.embopress.org/doi/full/10.15252/msb.20198994). The xiSEARCH scoring function is made up of several terms accounting for the goodness of fit of the spectra to the peptide pair selected from the database, including fragment mass error, percentage intensity explained, number of fragments, number of crosslinked fragments.
+The xiSEARCH algorithm is described in detail in [Mendez, Fischer *et al.* Mol. Sys. Bio. 2019](https://www.embopress.org/doi/full/10.15252/msb.20198994). The xiSEARCH scoring function is 
+made 
+up of several terms 
+accounting for the goodness of fit of the spectra to the peptide pair selected from the database, including fragment mass error, percentage intensity explained, number of fragments, number of crosslinked fragments.
```
Comment (lutzfischer) [inline README.md:15]: Why the newlines?

File: README.md [hunk 2]
```diff
@@ -9,7 +9,14 @@ xiSEARCH is a search engine for crosslinking mass spectrometry (crosslinking MS)
...
+
+
+![Xialgorithm](https://i.imgur.com/lEfpS5F.png)
```
Comment (lutzfischer) [inline README.md:18]: I would rather create a `doc` subfolder  and link to an image there e.g.
[Xialgorithm](doc/algorithm.png)
or to an github upload
! [xi_search_strategy] (https://user-images.githubusercontent.com/6330440/202421641-69fa3021-f53e-4eb1-9923-c9c740aca0e2.jpg)
xi_search_strategy

File: README.md [hunk 3]
```diff
@@ -45,20 +54,20 @@ Normally, all searches are performed with 2 crosslinkers selected: the crosslink
...
+| BS2G (Small Scale)    | BS2G crosslinker, K/S/T/Y/Nterm to  K/S/T/Y/Nterm                                                |
+| SDA    | SDA crosslinker, K/S/T/Y/Nterm to any amino acid                                                 |
+| BS3 (Large Scale)    | BS3 crosslinker, K/S/T/Y/Nterm to  K/S/T/Y/Nterm. Also for DSS.                                  |
+| BS3 (Small Scale)    | BS3 crosslinker, K/S/T/Y/Nterm to  K/S/T/Y/Nterm. Also for DSS.                                  |
+| DSSO (Large Scale)    | DSSO crosslinker,  K/S/T/Y/Nterm to  K/S/T/Y/Nterm  with cleavable stub fragment identification  |
+| DSSO (Small Scale)    | DSSO crosslinker, K/S/T/Y/Nterm to  K/S/T/Y/Nterm with cleavable stub fragment                   |
+| EDC    | EDC crosslinker, K/S/T/Y, Nterm to E/D/Cterm. No modifications defined.                          |
+| DSBU (Large Scale)    | DSBU crosslinker, K/S/T/Y/Nterm with cleavable stub fragment                                     |
+| DSBU (Small Scale)    | DSBU crosslinker, K/S/T/Y/Nterm with cleavable stub fragment                               <br/> |
```
Comment (lutzfischer) [inline README.md:68]:
```suggestion
| DSBU (Small Scale)    | DSBU crosslinker, K/S/T/Y/Nterm with cleavable stub fragment                                      |
```

File: README.md (hunk 1 above)
Comment (grandrea) [reply README.md:15]: ah i don't know what happened i must have been clicking around

File: README.md (hunk 3 above)
Comment (lutzfischer) [reply README.md]: or if you want a seperation to the last two a new line?
```suggestion
| DSBU (Small Scale)    | DSBU crosslinker, K/S/T/Y/Nterm with cleavable stub fragment                                |
|                               |                                                 |
```
?

File: README.md (hunk 1 above)
Comment (grandrea) [reply README.md:15]: let me fix

File: README.md (hunk 2 above)
Comment (grandrea) [reply README.md:18]: is that github link already correct?

Comment (lutzfischer) [reply README.md:18]: it should be

File: README.md [hunk 4]
```diff
@@ -275,3 +287,30 @@ crosslinker results in 2 stubs (here for SDA):
      loss:CleavableCrosslinkerPeptide:MASS:0;Name:0
      loss:CleavableCrosslinkerPeptide:MASS:82.04186;NAME:S
 
+#### Changing or adding new entries to the graphical config interface
+The options in the dropdown menu of the interface may be edited according to your needs.
```
Comment (lutzfischer) [inline README.md:291]:
```suggestion
The options in the drop-down menus and lists in the interface may be edited according to your needs. 
```

File: README.md [hunk 5]
```diff
@@ -160,6 +165,7 @@ Below is a list of settings that can be configured in a text config and their de
 | topmgcpeaks:150                        | how many alpha peptide candidates will be considered to find beta peptide.                                                                                          | Yes                            |
 | topmgxhits:10                          | how many combinations of alpha and beta peptides will be considered for final scoring                                                                               | Yes                            |
 | MAX_MODIFICATION_PER_PEPTIDE:3         | limit on how many modifications to consider per peptide. Both variable and fixed modifications count against the limit                                              | Yes                            |
+| MAX_MODIFICATION_PER_PEPTIDE:3         | limit on how many modifications to consider per peptide. Both variable and fixed modifications count against the limit                                              | Yes                            |
```
Comment (lutzfischer) [inline README.md:163]: that looks duplicated
```suggestion |
```

File: README.md [hunk 6]
```diff
@@ -275,3 +281,30 @@ crosslinker results in 2 stubs (here for SDA):
...
 
+#### Changing or adding new entries to the graphical config interface
+The options in the dropdown menu of the interface may be edited according to your needs. 
+The "BasicConfigEntries.conf" contains all the selectable config values. In this file new entries 
+for crosslinker, enzymes, modifications and losses can be freely defined. The file contains sections for crosslinker,
+modifications, losses, ions,enzymes and custom settings. Each section has a short description on how to add new entries.
+
+Additionally there is a file "BasicConfig.conf" containing default values for settings not exposed in the interface.
+
+Both of these are default files that change the options in the dropdown menu of the interface.
```
Comment (lutzfischer) [inline README.md:287]: That is not quite true. Settings in BasicConfig.conf do not show up in the interface at all. So any setting there has no effect on the interface (unless you translate the basic-config into the text config) but affect the search as such

File: README.md [hunk 7]
```diff
@@ -275,3 +276,30 @@ crosslinker results in 2 stubs (here for SDA):
      loss:CleavableCrosslinkerPeptide:MASS:0;Name:0
      loss:CleavableCrosslinkerPeptide:MASS:82.04186;NAME:S
 
+#### Changing or adding new entries to the graphical config interface
+The options in the dropdown menu of the interface may be edited according to your needs. 
+The "BasicConfigEntries.conf" contains all the selectable config values. In this file new entries 
+for crosslinker, enzymes, modifications and losses can be freely defined. The file contains sections for crosslinker,
+modifications, losses, ions,enzymes and custom settings. Each section has a short description on how to add new entries.
```
Comment (lutzfischer) [inline README.md:283]:
```suggestion
The "BasicConfigEntries.conf" contains all the selectable config values. In this file new entries for crosslinker, enzymes, modifications and losses can be freely defined. The file contains sections for crosslinker, modifications, losses, ions,enzymes and custom settings. Each section has a short description on how to add new entries.
```

File: README.md [hunk 8]
```diff
@@ -275,3 +276,30 @@ crosslinker results in 2 stubs (here for SDA):
...
+
+Additionally there is a file "BasicConfig.conf" containing default values for settings not exposed in the interface.
+
+Both of these are default files that change the options in the dropdown menu of the interface.
+
+## running xiSEARCH from command line
+
+xiSEARCH may be launched from the command line specifying database and config file. Often, a config file is created 
+in the interface and then used in launching searches from command line, for example as cluster jobs.
```
Comment (lutzfischer) [inline README.md:292]:
```suggestion
xiSEARCH may be launched from the command line specifying database and config file. Often, a config file is created in the interface and then used in launching searches from command line, for example as cluster jobs.
```

File: README.md [hunk 9]
```diff
@@ -275,3 +276,30 @@ crosslinker results in 2 stubs (here for SDA):
...
+
+    java -Xmx60G -cp /path/to/XiSearch.jar rappsilber.applications.Xi --config=MyConfig.config --peaks=peakfile.mgf --fasta=database.fasta -output=MyOutput.csv --locale=en
+
+will launch a search on peakfile.mgf with database.fasta and MyConfig.conf and 60Gb of RAM. Command line options are 
+available
```
Comment (lutzfischer) [inline README.md:297]:
```suggestion
will launch a search on peakfile.mgf with database.fasta and MyConfig.conf and 60Gb of RAM. Command line options are available
```

File: README.md (hunk 4 above)
Comment (grandrea) [reply README.md]: fixed in new commit

File: README.md [hunk 10]
```diff
@@ -22,11 +23,10 @@ The scoring function is applied to explain each spectrum with without considerin
 
 
 ### The interface
-The interface provides several tabs. The first two tabs are the main ones for configuring the search. The first one (Files) defines the input and output, i.e. the the peaklist- and fasta-files to be search and where to write the result. 
-The second one (Parameters) configures the actual search. The third one (Feedback) provides the log of the current search and provides a means to contact the developers. The fourth tab contains the change log/Version history. 
+The interface provides several tabs. The first two tabs are the main ones for configuring the search. The first one (Files) defines the input and output, i.e. the peaklist and fasta files to be search and where to write the result. The second one (Parameters) configures the actual search. The third one (Feedback) provides the log of the current search and provides a means to contact the developers. The fourth tab contains the change log/Version history.
```
Comment (lutzfischer) [inline README.md:26]: Sorry my error
```suggestion
The interface provides several tabs. The first two tabs are the main ones for configuring the search. The first one (Files) defines the input and output, i.e. the peaklist and fasta files to be search and where to write the result. The second one (Parameters) configures the actual search. The third one (Feedback) provides the log of the current search and provides a means to contact the developers. The fourth tab contains the change log/version history. 
```

File: README.md [hunk 11]
```diff
@@ -143,35 +142,40 @@ multiple options or settings are separated with a comma.
 
 Below is a list of settings that can be configured in a text config and their description.
 
+All possible options and their default values are also found in the file 
+"src/main/java/rappsilber/data/csv/BasicConfigEntries.
+conf".
```
Comment (lutzfischer) [inline README.md:147]:
```suggestion
All possible options and their default values are also found in the BasicConfigEntries.conf file.
```

File: README.md [hunk 12]
```diff
@@ -239,39 +244,64 @@ noncovalent modifications, including the additional "NonCovalent" crosslinker wi
...
+
+xiSEARCH may be launched from the command line specifying database and config file. Often, a config file is created in the interface and then used in launching searches from command line, for example as cluster jobs.
+
+    java -Xmx60G -cp /path/to/XiSearch.jar rappsilber.applications.Xi --config=MyConfig.config --peaks=peakfile.mgf --fasta=database.fasta -output=MyOutput.csv --locale=en
+
+will launch a search on peakfile.mgf with database.fasta and MyConfig.conf and 60Gb of RAM. Command line options are available
+
+    java -cp /path/to/XiSearch.jar rappsilber.applications.Xi --help
+
+If there is more than one peaklist to be searched, the .mgf files should be zipped together and the zip file should be given as the option of --peaks .
```
Comment (lutzfischer) [inline README.md:303]:
```suggestion
If there is more than one peaklist to be searched, the .mgf files can either be zipped together and the zip file be given as the option of --peaks= or several --peaks= options can be given.

Also several fasta files can be given, by providing  a --fasta= argument per fasta file
```

**Pull request merged**